### PR TITLE
Logging errors in celery tasks

### LINKED ISSUE #5498: Set up logging correctly
Currently, logging is hardcoded in the app and never really used. Logging should be used almost on every action with proper label as:

- trace
- debug
- info
- warn
- error

Right now, logger is hardcoded to show ERROR only. This needs to be configurable through the command line. 

Also, I think default should be `INFO` or even `DEBUG` for the app itself and warn for other 3rd party libraries.

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Related to #5498 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Only a handful of celery tasks were able to log information about any errors which may occur. Extended to all methods.

#### Changes proposed in this pull request:

- Logging celery task errors

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Remove fixes from the description. Also, remove print statements. And check that handled exceptions are warning, only actual unhandled exceptions are marked as errors

**Comment by mrsaicharan1** [comment]
Added the latest updates and appropriate logging for handled exceptions done(warnings for handled exceptions)

File: app/api/helpers/tasks.py [hunk 1]
```diff
@@ -190,8 +189,8 @@ def export_ical_task(self, event_id, temp=True):
             save_to_db(event)
 
     except Exception as e:
-        print(traceback.format_exc())
         result = {'__error': True, 'result': str(e)}
+        logging.warning('Error in ical download')
```
Comment (iamareebjamal) [inline app/api/helpers/tasks.py]: It's not a handled exception unless we have some fallback to do in the block

Comment (mrsaicharan1) [reply app/api/helpers/tasks.py]: @ iamareebjamal Alright. Logging on appropriate levels done. Can you check if it's right?

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 PLease review :)

**Pull request merged**